dist/ventus.js

```diff
@@ -1088,7 +1088,7 @@ define('ventus/core/view', ['$'], function ($) {
             });
         },
         onAnimationEnd: function (callback, scope) {
-            this.one(animationEventNames, function () {
+            this.on(animationEventNames, function () {
```

Comment:
This will lead to memory leaks since the event listener is never unregistered.

Reply:
Hi, thanks for reviewing, I just made an update.